src/structures/GuildEmoji.js

```diff
@@ -45,6 +45,13 @@ class GuildEmoji extends Emoji {
      */
     if (typeof data.managed !== 'undefined') this.managed = data.managed;
 
+    /**
+     * Whether this emoji is available, only present on premium guilds
```

Comment:
All emoji objects have this property, whether the guild is premium or not

Reply:
It doesn't seem to be present on guilds that don't have the Nitro Server Boost feature enabled, but I think it's safe to assume that it will land on all guilds eventually so I'll go ahead and remove that note